### PR TITLE
[SKYRAT CLEANUP] Removes / updates all bubber comments in the skyrat modular file as we are no longer their subject

### PR DESCRIPTION
**Pull request by Majkl-J**
## About The Pull Request

"I'm going to extend my break" - Me, 2 days ago

"Hey I should start cleaning up the Skyrat modular folder" - Me, Today

I yet again fall victim to my own hubris

Left out some BUBBER TODOs, feel free to do these if you find them manageable

## Why It's Good For The Game

We made some old mistakes with some edits (Like commenting out ruins instead of making them unpickable for the ruin system) (This I fixed, admins can now bus them in!)

We also made a lot of edits to Skyrat code, a lot of which can be cleanly merged into their code now

## Proof Of Testing

If it compiles, it works.

Unlike my modularization PR from a year ago this is less fuckupable

## Changelog

N/A, not player facing

### CODE REVIEW

**Comment by ReturnToZender** [comment]
I kinda like this

**Review by lessthnthree — APPROVED** [review]

**Comment by lessthnthree** [comment]
I can't wait for every other PR to go into merge conflict when we push button.

**Comment by Majkl-J** [comment]
I push button

**Pull request merged**
